### PR TITLE
tmp_path is required if user deploy tiflash instance other than tiup/tidb-operator

### PR DESCRIPTION
**Pull request by JaySon-Huang**
Signed-off-by: JaySon-Huang <tshent@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

The configuration "tmp_path" in TiFlash config file is required if the user deploy a tiflash instance other than tiup/tidb-operator

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

master/6.0 are not affected.
5.4/5.3/5.2/5.1/5.0/4.0 are affected.
Note that this PR is base on release-5.4

- [ ] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tics/issues/4091

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ delta_index_cache_size = 0
 # path_realtime_mode = false
 
 ## TiFlash 临时文件的存放路径。默认使用 [`path` 或者 `storage.latest.dir` 的第一个目录] + "/tmp"
```
Comment (flowbehappy) [inline tiflash/tiflash-configuration.md:57]:
```suggestion
## TiFlash 临时文件的存放路径。通常使用 [`path` 或者 `storage.latest.dir` 的第一个目录] + "/tmp"
```

**Review by flowbehappy — APPROVED** [review]

File: tiflash/tiflash-configuration.md (hunk 1 above)
Comment (TomShawn) [reply tiflash/tiflash-configuration.md:57]: 默认的路径是什么呢？

Comment (flowbehappy) [reply tiflash/tiflash-configuration.md:57]: v6.0 以前版本必须配置这个选项，所以没有默认路径。 6.0 及以后才有默认路径

**Review by TomShawn — APPROVED** [review]
PTAL @flowbehappy 's comment

File: tiflash/tiflash-configuration.md (hunk 1 above)
Comment (JaySon-Huang) [reply tiflash/tiflash-configuration.md:57]: 必需配置这个选项主要是一个 bug 导致的 https://github.com/pingcap/tics/issues/4091。在一些后续的小版本上会修复。

如果用户使用 tiup/tidb-operator 部署的话，用户不需要手动设置，tiup/tidb-operator 会生成一个默认值。所以大部分用户都不需要关注。这个改动主要是给需要手动部署 tiflash 节点的用户（一般很少）。

**Review by TomShawn — APPROVED** [review]

**Pull request merged**